app/assets/stylesheets/posts.scss

```diff
@@ -240,4 +240,8 @@
   a {
     color: black;
   }
+}
+
+.textarea{
+  inline-size: 700px;
```

Comment:
can you add % instead of px? px doesn't work well for all the screen since it is specific to a particular screen resolution